src/hazelcore/Action_common.re

```diff
@@ -304,16 +324,68 @@ let complete_tuple_ =
         fun
         | (rev_holes, u_gen) => (rev_holes |> List.rev, u_gen)
       );
-    (
-      List.fold_right(
-        (new_hole, suffix: Seq.affix(_)) => A(comma, S(new_hole, suffix)),
-        new_holes,
-        Seq.E,
-      ),
-      u_gen,
+    print_endline("first_seq_length = " ++ string_of_int(first_seq_length));
```

Comment:
don't include debug prints in dev

Reply:
@cyrus- Pushed a commit that removed all debug prints in this PR.